README.md

```diff
@@ -33,6 +33,28 @@ The software is provided under [Apache-2.0 license](LICENSE). Contributions to t
 
 This project contains code from other projects. The original license text is included in those source files. They must comply with our [license guide](https://os.mbed.com/docs/latest/reference/license.html)
 
+Folders containing files not under Apache 2.0 as listed above. The original license text is included in those source files.
+
+- cmsis - MIT/BSD-3-Clause
+- components/802.15.4_RF/mcr20a-rf-driver - BSD-3-Clause
+- features/cryptocell/FEATURE_CRYPTOCELL310 - BSD-style
+- features/FEATURE_BOOTLOADER - PBL
+- features/FEATURE_BLE/targets - BSD-style/PBL/MIT
+- features/lorawan - Revised BSD
+- features/lwipstack - BSD-3-Clause/MIT
+- features/nanostack/sal-stack-nanostack - BSD-3-Clause
```

Comment:
This is only partially true..
Most of the Nanostack itself is Apache 2.0, but there are Thread related code which is BSD.
https://github.com/ARMmbed/mbed-os/tree/master/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread

Reply:
I would recommend doing a proper table with links to the sources/licence file in the three. Makes it easier to find those, since for the BSD related stuff - the end product needs to show the license SOMEWHERE in the UI/manuals.

Reply:
I stated this lists contains folders with non-apache files (thus nanostack folder marked as it contains BSD-3-Clause). Each of these folders should have then list of non Apache files.

I like the idea having the tree link

> features/nanostack/sal-stack-nanostack - BSD-3-Clause

This would have url link so we can go to nanostack folder and there would be: 
- license file (in this case BSD-3-Clause)
- Readme with license information (what files are non-apache)

Reply:
Updated.  The latest push contains links to folders. Each of them should contain Readme (will be added later by separate PRs). I'll add this requirement to our documentation first and will request each module to add it later.

Readme.md in the root should just point to `features/nanostack` (license overview). 
`features/nanostack/Readme.md` will contain list of files/modules inside with licenses (detailed view).

Reply:
Our license page actually contains this, we just need to fix the current ones and check the new ones as a requirement

https://os.mbed.com/docs/mbed-os/v5.11/contributing/license.html (`Using a different license`).